### PR TITLE
Updated sample to MSAL 0.1.4 and did some minor code cleanup

### PR DESCRIPTION
**Pull request by oldalton**
* Updated sample from 0.1.3 to 0.1.4 (users changed to accounts)
* I pointed it to release/0.1.4 branch for now until we merge it back to master and tag. Will update pod file when it happens
* I did some minor cleanup for the callGraphButton method - I separated it into multiple methods for easier readability. I also fixed alignment in multiple places, so it will show many changes, although there're not that many. 
* I moved viewDidLoad to the beginning of the file to show PublicClientApplication creation in the very beginning.
* Looking forward to feedback for those changes.

### CODE REVIEW

**Comment by oldalton** [comment]
@unpluggedk, @jasoncoolmax, @antrix1989, @rohitnarula7176, @xerners 
I can't add reviewers here, so tagging you to provide feedback.

File: MSALiOS/ViewController.swift [hunk 1]
```diff
@@ -33,196 +33,232 @@ import MSAL
...
+                                    it is of the form https://<instance/<tenant>, where <instance> is the
+                                    directory host (e.g. https://login.microsoftonline.com) and <tenant> is a
+                                    identifier within the directory itself (e.g. a domain associated to the
+                                    tenant, such as contoso.onmicrosoft.com, or the GUID representing the
+                                    TenantID property of the directory)
+             - error                The error that occurred creating the application object, if any, if you're
+                                    not interested in the specific error pass in nil.
+             */
+
+            self.applicationContext = try MSALPublicClientApplication.init(clientId: kClientID, authority: kAuthority)
```
Comment (antrix1989) [inline MSALiOS/ViewController.swift]: nit: we can skip `.init` here, just write: `MSALPublicClientApplication(clientId: kClientID, authority: kAuthority)`

File: MSALiOS/ViewController.swift [hunk 2]
```diff
@@ -33,196 +33,232 @@ import MSAL
...
+        } catch let error {
+            self.loggingText.text = "Unable to create Application Context \(error)"
+        }
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+
+        if self.accessToken.isEmpty {
+
+            signoutButton.isEnabled = false
```
Comment (antrix1989) [inline MSALiOS/ViewController.swift]: nit: I would simplify it to: `signoutButton.isEnabled = accessToken.isEmpty`

File: MSALiOS/ViewController.swift (hunk 1 above)
Comment (oldalton) [reply MSALiOS/ViewController.swift]: yeah, I saw it today, it was copied from the old code :) Will fix.

File: MSALiOS/ViewController.swift [hunk 3]
```diff
@@ -33,196 +33,232 @@ import MSAL
...
+    func currentAccount() -> MSALAccount? {
+
+        // We retrieve our current account by getting the first account from cache
+        // In multi-account applications, account should be retrieved by home account identifier or username instead
+
+        do {
+
+            let cachedAccounts = try self.applicationContext!.accounts()
+
+            guard cachedAccounts.isEmpty else {
```
Comment (antrix1989) [inline MSALiOS/ViewController.swift]: I would prefer `if !cachedAccounts.isEmpty { return cachedAccounts.first }`.

File: MSALiOS/ViewController.swift [hunk 4]
```diff
@@ -33,196 +33,232 @@ import MSAL
...
+
+        URLSession.shared.dataTask(with: request) { data, response, error in
         
         let result = try? JSONSerialization.jsonObject(with: data!, options: [])
-                    if result != nil {
-                
-                self.loggingText.text = result.debugDescription
+
+            if result != nil {
+                self.updateLogging(text: result.debugDescription)
```
Comment (antrix1989) [inline MSALiOS/ViewController.swift]: it will add "Optional(..." to the string, I would do `if let result = result` here.

**Pull request closed without merging**